catalog/spatial/geocoding/spatial-geocoding-extract/src/main/java/org/codice/ddf/spatial/geocoding/extract/GeoNamesFileExtractor.java

```diff
@@ -161,7 +174,7 @@ private InputStream getInputStreamFromResource(
       } else if (resource.matches("((?i)cities[0-9]+)")) {
         resource = resource.toLowerCase();
         // Support case insensitive country codes
-      } else if (!resource.equalsIgnoreCase("allCountries")) {
+      } else if (!resource.equalsIgnoreCase("allCountries") && !resourceIsValidUrl) {
```

Comment:
✏️ Is this `&& !resourceIsValidUrl` needed here?

Reply:
Yes, otherwise valid URLs become uppercase. I had to work with the existing logic.

Reply:
Isn't it a guarantee that all uppercase/lowercase variations of the String "allCountries" are not valid URLs?

Reply:
I think the condition is evaluating if the string is not "allCountries" (regardless of capitalization). If the string is not "allCountries" it is not guaranteed be a valid URL.

Reply:
🤦‍♀️ You're right. I misread that.